pkg/handlers/kas-grants.go

```diff
@@ -98,13 +99,17 @@ func (h Handler) DeleteKasGrantFromNamespace(ctx context.Context, ns_id string,
 	return resp.GetNamespaceKeyAccessServer(), nil
 }
 
-func (h Handler) ListKasGrants(ctx context.Context, kas_id, kas_uri string) ([]*kasregistry.KeyAccessServerGrants, error) {
+func (h Handler) ListKasGrants(ctx context.Context, kas_id, kas_uri string, limit, offset int32) ([]*kasregistry.KeyAccessServerGrants, *policy.PageResponse, error) {
```

Comment:
Why does this function use a passed in context vs the others that use `h.ctx`?

Reply:
Great question, and none of them should actually be using a struct context: https://go.dev/blog/context-and-structs 

We have an open issue to refactor this anti-pattern throughout, which has not been prioritized: https://github.com/opentdf/otdfctl/issues/364

Reply:
Ah got it, thanks for the link